safe/gis/generic_expressions.py

```diff
@@ -230,3 +232,39 @@ def beautify_time(inasafe_time, feature, parent):
         inasafe_time, '%Y-%m-%dT%H:%M:%S.%f')
     time = datetime_object.strftime('%H:%M')
     return time
+
+
+description = tr(
+    "Given a keyword, it will return the value of the keyword "
+    "from the hazard layer's extra keywords.")
+examples = {
+    "hazard_extra_keyword( 'depth' )": tr(
```

Comment:
@myarjunar @ismailsunni Is `depth` existing? I can't find any reference in inasafe ...

Reply:
yes, it is. It's a part of the new `extra_keywords` property in the keywords.

Reply:
Yes I saw your message in gitter after that. I thought it was related to flood depth, so I couldn't understand why this keyword (because it would be a kind of dictionary). But for EQ realtime, yes it makes sense!

Reply:
Ah, sorry for late replying your comment here :D